frontend/server/src/DAO/Problems.php

```diff
@@ -1157,4 +1157,18 @@ public static function hasBeenUsedInCoursesOrContests(
         /** @var list<array{name: string, problems_per_tag: int}> */
         return \OmegaUp\MySQLConnection::getInstance()->GetAll($sql);
     }
+
+    final public static function getProblemAliasRandom(): string {
+        $sql = 'SELECT
+                    alias
+                FROM
+                    Problems
+                WHERE
+                    quality_seal = 1
+                GROUP BY
+                    rand() LIMIT 1;';
```

Comment:
nit:

```suggestion
                    RAND() LIMIT 1;';
```

todas las palabras reservadas y funciones de SQL van en mayúsculas. En [la documentación](https://dev.mysql.com/doc/refman/8.0/en/mathematical-functions.html#function_rand) así aparecen, por ejemplo.